### ISSUE TITLE
Missing libpopt dependency

### ISSUE THREAD

**Issue by ericwongcm** [mikebrady/shairport-sync#67]
If I build Shairport-sync for Openwrt using any version above 2.1.12, I will get an error which basically says libpopt is missing even though this package is already selected.

I think this is related to the code change in version 2.1.13 that you have implemented for checking dependencies.

Take note that I am able to build shairport-sync version 2.1.12 for Openwrt without any problems.
Btw, shairport-sync for openwrt have not been updated to newer versions since 2.1.12.

**Comment by mikebrady** [comment]
Hi Eric. It's true that I've neglected OpenWrt for a while. However, thanks to Ted Hess  there is now a shairport-sync build package in OpenWrt trunk, (which has an updated libpopt package), and the package works up to 2.1.15 at least. Another issue is that there are more options now in Shairport Sync, and the startup script for OpenWrt doesn't expose them. To get around this, I'm starting to add support for a settings file /etc/shairport-sync.conf. This will work in OpenWrt as well as Debian, Raspbian, Ubuntu, etc.

**Comment by ericwongcm** [comment]
I see... I have tried the shairport-sync build package in OpenWrt trunk but that didn't work for Attitude Adjustment. I still run into the same error....

Since you mentioned there is an updated libpopt package too, I tried to manually update the libpopt package in Attitude Adjustment to the same package used in trunk but that results in some gettext version related error that I can't fix... 

Without testing, I hope the shairport-sync build package in OpenWrt trunk will work for trunk.
Anyway, I guess I will stay at 2.1.12 for now.

Thanks
